01 Operator Overloading/OperatorOverloading.Model/Money.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OperatorOverloading.Model
+{
+    public class Money
+    {
+        //Double Parameter constructor
+        public Money(double amount, string currency)
+        {
+            if (double.IsInfinity(amount))
+            {
+                throw new System.ArgumentException(Messages.AmountOverflow);
+            }
+            this.Amount = amount;
+            this.Currency = currency;
+        }
+
+        //Single Parameter Constructor
+        public Money(string input)
+        {
+            if (string.IsNullOrWhiteSpace(input))
+            {
+                throw new System.ArgumentException(Messages.NullInput);
+            }
+
+            var inputarr = input.Split(' ');
```

Comment:
It will throw exception since the string inputarr[1] will be empty and it throws exception